bukuserver/forms.py

```diff
@@ -1,44 +1,44 @@
 """Forms module."""
 # pylint: disable=too-few-public-methods, missing-docstring
 from typing import Any, Dict, Tuple
-from textwrap import dedent
 from flask_wtf import FlaskForm
 from wtforms.fields import BooleanField, FieldList, StringField, TextAreaField, HiddenField
 from wtforms.validators import DataRequired, InputRequired, ValidationError
 from buku import DELIM, parse_tags
+from bukuserver import _, _l, LazyString
 from bukuserver.response import Response
 
 def validate_tag(form, field):
     if not isinstance(field.data, str):
-        raise ValidationError('Tag must be a string.')
+        raise ValidationError(_('Tag must be a string.'))
     if DELIM in field.data:
-        raise ValidationError('Tag must not contain delimiter \"{}\".'.format(DELIM))
+        raise ValidationError(_('Tag must not contain delimiter "%(delim)s".', delim=DELIM))
 
 
 class SearchBookmarksForm(FlaskForm):
-    keywords = FieldList(StringField('Keywords'), min_entries=1)
-    all_keywords = BooleanField('Match all keywords', default=True, description='Exclude partial matches (with multiple keywords)')
-    markers = BooleanField('With markers', default=True, description=dedent('''\
-        The search string will be split into multiple keywords, each will be applied to a field based on prefix:
-         - keywords starting with '.', '>' or ':' will be searched for in title, description and URL respectively
-         - '#' will be searched for in tags (comma-separated, partial matches; not affected by Deep Search)
-         - '#,' is the same but will match FULL tags only
-         - '*' will be searched for in all fields (this prefix can be omitted in the 1st keyword)
-        Keywords need to be separated by placing spaces before the prefix.
-    '''))
-    deep = BooleanField('Deep search', description='When unset, only FULL words will be matched.')
-    regex = BooleanField('Regex', description='The keyword(s) are regular expressions (overrides other options).')
+    keywords = FieldList(StringField(_l('Keywords')), min_entries=1)
+    all_keywords = BooleanField(_l('Match all keywords'), default=True, description=_l('Exclude partial matches (with multiple keywords)'))
+    markers = BooleanField(_l('With markers'), default=True, description=LazyString(lambda: '\n'.join([
+        _('The search string will be split into multiple keywords, each will be applied to a field based on prefix:'),
+        _(" - keywords starting with '.', '>' or ':' will be searched for in title, description and URL respectively"),
+        _(" - '#' will be searched for in tags (comma-separated, partial matches; not affected by Deep Search)"),
+        _(" - '#,' is the same but will match FULL tags only"),
+        _(" - '*' will be searched for in all fields (this prefix can be omitted in the 1st keyword)"),
+        _('Keywords need to be separated by placing spaces before the prefix.'),
+    ])))
```

Comment:
Splitting these into separate lines and joining immediately at the time of printing is more convenient than trying to deal with the whole block of text as a single message key.